aten/src/ATen/native/mps/MPSTensorImpl.cpp

```diff
@@ -0,0 +1,29 @@
+//  Copyright © 2022 Apple Inc.
+
+#include <ATen/native/mps/MPSTensorImpl.h>
+#include <c10/core/DispatchKey.h>
+#include <c10/core/TensorImpl.h>
+#include <c10/util/ArrayRef.h>
+
+namespace at {
+namespace native {
+namespace mps {
+
+using at::Device;
+using at::DeviceType;
+
+const char* MPSTensorImpl::tensorimpl_type_name() const {
+  return "MPSTensorImpl";
+}
+
+const at::Storage& MPSTensorImpl::storage() const {
```

Comment:
This one doesn't actually need to be overriden right?

Reply:
We are not using MPSTensorImpl and using the default c10::TensorImpl.